examples/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -0,0 +1,139 @@
+  }
+
+  /**
+   * Construct our gRPC client that connects to the pubsub server using an existing channel.
+   *
+   * @param channel    channel that has been built already
+   * @param callCredentials  the Google call credentials created from a JSON file
+   */
+  GoogleAuthClient(ManagedChannel channel, CallCredentials callCredentials) {
+    this.channel = channel;
+    blockingStub = PublisherGrpc.newBlockingStub(channel).withCallCredentials(callCredentials);
+  }
+
+  public void shutdown() throws InterruptedException {
+    channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
+  }
+
+  /**
+   * Get topics (max 10) for our project ID: the topic list is logged to the logger.
+   *
+   * @param projectID  the GCP project ID to get the pubsub topics for. This is a string like
+   *                   "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
+   *                   the project ID for the project you created.
+   */
+  public void getTopics(String projectID) {
+    logger.info("Will try to get topics for poroject " + projectID + " ...");
+
+    ListTopicsRequest request = ListTopicsRequest.newBuilder()
+            .setPageSize(10)        // get max 10 topics
+            .setProject(projectID)  // for our projectID
+            .build();
+
+    ListTopicsResponse response;
+    try {
+      response = blockingStub.listTopics(request);
+    } catch (StatusRuntimeException e) {
+      logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
+      return;
+    }
+    logger.info("Topics list:\n" + response.getTopicsList());
+  }
+
+  /**
+   * The app requires 2 arguments as described in
+   * @see <a href="../../../../../../GOOGLE_AUTH_EXAMPLE.md">Google Auth Example README</a>
+   *
+   * arg0 = location of the JSON file for the service account you created in the GCP console
+   * arg1 = project name in the form "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
+   *        the project ID for the project you created.
+   *
+   */
+  public static void main(String[] args) throws Exception {
+    if (args.length < 2) {
+      logger.severe("Usage: please pass 2 arguments:\n" +
+                    "arg0 = location of the JSON file for the service account you created in the GCP console\n" +
+                    "arg1 = project name in the form \"projects/xyz\" where \"xyz\" is the project ID of the project you created.\n");
+      System.exit(1);
+    }
+    GoogleCredentials credentials = GoogleCredentials.fromStream(new FileInputStream(args[0]));
+
+    // We need to create appropriate scope as per https://cloud.google.com/storage/docs/authentication#oauth-scopes
+    credentials = credentials.createScoped(Arrays.asList("https://www.googleapis.com/auth/cloud-platform"));
+
+    // credentials must be refreshed before the access token is available
+    credentials.refreshIfExpired();
+    credentials.getAccessToken();
+    GoogleAuthClient client = new GoogleAuthClient("pubsub.googleapis.com", 443,
```

Comment:
same comment here.   Prefer to break at `new GoogleAuthClient(` or on `=`, to keep as much of the pieces together.